tests/modules/lint.py

```diff
@@ -82,6 +82,16 @@ def test_modules_lint_multiple_remotes(self):
     assert len(module_lint.warned) >= 0
 
 
+def test_modules_lint_registry(self):
```

Comment:
Can we also have a test where it runs successfully with an alternative registry? Just to be sure we don't break this by accident.

Reply:
ah, sorry I read the test wrong